app/config/collections.php

```diff
@@ -1366,7 +1377,545 @@
+                'orders' => [],
+            ],
+            [
+                '$id' => ID::custom('_key_targetInternalId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['targetInternalId'],
+                'lengths' => [],
+                'orders' => [],
+            ],
+            [
+                '$id' => ID::custom('_key_topicId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['topicId'],
+                'lengths' => [],
+                'orders' => [],
+            ],
+            [
+                '$id' => ID::custom('_key_topicInternalId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['topicInternalId'],
+                'lengths' => [],
+                'orders' => [],
+            ]
+        ],
+    ],
+
+    'targets' => [
+        '$collection' => ID::custom(DATABASE::METADATA),
+        '$id' => ID::custom('targets'),
+        'name' => 'Targets',
+        'attributes' => [
+            [
+                '$id' => ID::custom('userId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('userInternalId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('providerId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('providerInternalId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('provider'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => ['subQueryProvider'],
+            ],
```

Comment:
Let's remove this (as well as the sub-query), we can manually get the provider by ID when needed

Reply:
done this